src/services/__tests__/models/Response.test.ts

```diff
@@ -31,5 +31,12 @@ describe('Models', () => {
       const resp = new ResponseModel(parser, 'default', true, {}, opts);
       expect(resp.type).toEqual('error');
     });
+
+    test('should be error if showExtensions is true', () => {
```

Comment:
should be error?

Reply:
Oops!  I missed updating the first half.  ;)